src/js/script.js

```diff
@@ -0,0 +1,10 @@
+var navMain = document.querySelector('.nav');
+var navToggle = document.querySelector('.nav__toggle');
+
+navToggle.addEventListener('click', function() {
+if (navMain.classList.contains('nav--open')) {
+navMain.classList.remove('nav--open');
+} else {
+navMain.classList.add('nav--open');
+}
```

Comment:
Align the code.

For toggling a class name, you can use the [toggle method](https://code.mu/ru/javascript/manual/dom/classList/toggle/). It simplifies your code and improves readability.